Package.swift

```diff
@@ -1,13 +1,17 @@
+// swift-tools-version:5.1
+
 import PackageDescription
 
 let package = Package(
   name: "UXKit",
-	
```

Comment:
Aren't the excludes necessary to build w/o warnings? 🤔
Also, we have a specific 5.1 Package manifest. Ah, maybe they haven't been necessary w/ 5.1 but are with newer Swift versions? (when SPM gained resource support).